src/macrofier/engine.mjs

```diff
@@ -625,21 +625,36 @@ function generateSchemas(json, templates, options) {
     results.push(result)
   }
 
+  const list = []
+
   // schemas may be 1 or 2 levels deeps
   Object.entries(schemas).forEach( ([name, schema]) => {
     if (isSchema(schema)) {
-      generate(name, schema)
+      list.push([name, schema])
     }
     else if (typeof schema === 'object') {
```

Comment:
Not directly related to your change, but I think we can get rid of this entire `else` block. It's for external schemas, that won't get output by the current module. Your code below doesn't account for these, but we actually filter them out immediately up at the top of the code after calling this method

Probably makes sense to leave your sorter method the way it is, and remove this else block.